Assignment/starter_code/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,69 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function(){
```

Comment:
An honest mistake, but look closely, they're asking to hide the _`header`_, not the `h1` — you got the concept down correctly though! :)